src/main/converter/opds.ts

```diff
@@ -49,14 +49,16 @@ export class OpdsFeedViewConverter {
     public convertOpdsLinkToView(link: OPDSLink, url: string): OpdsLinkView {
         // Title could be defined on multiple lines
         // Only keep the first one
-        let title = link.Title;
-        const titleParts = title.split("\n");
-        title = titleParts[0];
+        const titleParts = link.Title.split("\n").filter((text) => text);
+        const title = titleParts[0].trim();
+        const subtitle = titleParts[1] && titleParts[1].trim();
```

Comment:
if `subtitle` == `title`, then set `subtitle` to `undefined`, otherwise there are duplicate UI labels:
![ThoriumOPDSsubtitle](https://user-images.githubusercontent.com/625201/67391941-9169af00-f597-11e9-8ebe-bee76727d9b8.png)

Reply:
well seen, fixed